edx_sga/sga.py

```diff
@@ -563,7 +652,7 @@ def load_resource(resource_path):  # pragma: NO COVER
 
 def render_template(template_path, context={}):  # pragma: NO COVER
```

Comment:
This is one of things we really need to fix as it is potentially dangerous.  An easy way to fix is to set `context=None` in the function and below add:
```python
if context is None:
    context = {}
```